config/rbac/leader_election_role_binding.yaml

```diff
@@ -8,5 +8,5 @@ roleRef:
   name: leader-election-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: nexus-operator-sa
```

Comment:
this should be `name: sa` with no `namespace` (`kustomize` will add the correct namespace)